### ISSUE TITLE
why in adapt.py  line 91  label_tgt = make_variable(torch.ones(feat_tgt.size(0)).long())?  I think  label_tgt = make_variable(torch.zeros(feat_tgt.size(0)).long())

### ISSUE THREAD

**Issue by zzzpc** [corenel/pytorch-adda#28]

**Comment by zzzpc** [comment]
cause in line 63   label_tgt = make_variable(torch.zeros(feat_tgt.size(0)).long())
why in line 91 label_tgt changes to  torch.ones(feat_tgt.size(0)).long() ?

**Comment by GuoJingtao-1997** [comment]
In my view, the target encoder aims to map the target features into the source domain so that we can use the pre-trained source classifier to classify them. Therefore, we would like to make the target label after discriminating close to the source label that is one.
